src/pages/beer-info/Index.jsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import fetchFromParams from '../../domain/fetchAPI/fetchFromParams';
+import { BREWDOG_API } from '../../constants/apiUrl';
+import genericBottle from '../../assets/generic-bottle.png';
+
+export default function BeerInfo() {
+  const [beer, setBeer] = useState(null);
+  const { id } = useParams();
+
+  useEffect(() => {
+    const fetchBeer = async () => {
+      const data = await fetchFromParams(BREWDOG_API, `/${id}`);
+      setBeer(data);
```

Comment:
Did you account for fact data can be undefined? (if API request fails for any reason)

Reply:
Also, if this api returns array, I would suggest to do `setBeer(data[0])` because it is easier to manipulate later